src/common/util.ts

```diff
@@ -50,3 +50,8 @@ export const iterate = <T>(
   }
   return res;
 };
+
+/** helper to put a delay in JS */
```

Comment:
unrelated to this ticket, thought I'd sneak in this helper, mostly used by FE to delay calls to check loading states

Reply:
https://github.com/SeedCompany/cord-api-v3/blob/a3afcbe5eb4660d4c9d53874505e85034b9b2bce/src/common/util.ts#L9-L10